client/app.jsx

```diff
@@ -12,31 +12,53 @@ export default class App extends React.Component {
     super(props);
     this.state = ({
       user: null,
+      newUser: false,
       isAuthorizing: true,
       route: parseRoute(window.location.hash)
     });
     this.handleSignIn = this.handleSignIn.bind(this);
     this.handleSignOut = this.handleSignOut.bind(this);
+    this.endTour = this.endTour.bind(this);
   }
 
   handleSignIn(result) {
     const { user, token } = result;
+    const { newUser } = jwtDecode(token);
     window.localStorage.setItem('user-token', token);
-    this.setState({ user, token });
+    window.localStorage.setItem('newUser', newUser);
+    this.setState({ user, token, newUser });
   }
 
   handleSignOut() {
     window.localStorage.removeItem('user-token');
     this.setState({ user: null });
   }
 
+  endTour() {
+    const { user } = this.state;
+    const req = {
+      method: 'PATCH',
+      headers: {
+        'Content-Type': 'application/json',
```

Comment:
You can omit the `content-type` header here, as it's meant to only be present in HTTP messages with a body

Reply:
Thanks for the clarification! I removed line 42

![image](https://user-images.githubusercontent.com/109567499/202825875-4b6cee77-9cfa-4d16-b930-2bb733bc7225.png)